src/arsciexp.py

```diff
@@ -11,6 +11,7 @@
 import arload
 import arcomb
 import arflux
+import armasters
```

Comment:
I started walking down this path a bit and realized it might require quite
a bit of modifying.    And I actually prefer the current approach.   If the user
wants to specify a specific bias (or flat), then they use the 

reduce usebias name.fits

as you implemented.  If they want to use the Masters, they do that.  If
they want to remake a bias, they can do that.  If they want to use only
specific Master frames, they can specify those with 

reduce usebias xxx.fits

Hmm..  I see one problem:  multiple detectors.  
We're not setup for that with the old way..